Makefile

```diff
@@ -19,14 +21,37 @@ install-icons:
 	cp icons/qappmenu-power.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-power.svg
 	cp icons/qappmenu-qube.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-qube.svg
 	cp icons/qappmenu-settings.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-settings.svg
+	cp icons/qappmenu-bookmark-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-bookmark-black.svg
+	cp icons/qappmenu-bookmark-white.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-bookmark-white.svg
+	cp icons/qappmenu-bookmark-fill-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-bookmark-fill-black.svg
+	cp icons/qappmenu-bookmark-fill-white.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-bookmark-fill-white.svg
+	cp icons/qappmenu-list-white.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-list-white.svg
+	cp icons/qappmenu-list-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-list-black.svg
+	cp icons/qappmenu-grid-white.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-grid-white.svg
+	cp icons/qappmenu-grid-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-grid-black.svg
+	cp icons/qappmenu-sun-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-sun-black.svg
+	cp icons/qappmenu-sun-white.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-sun-white.svg
+	cp icons/qappmenu-moon-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-moon-black.svg
+	cp icons/qappmenu-services.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-services.svg
+
+update-icons:
+	gtk-update-icon-cache $(DESTDIR)/usr/share/icons/hicolor
```

Comment:
This should be called at install time, not build time.
See how it's done in desktop-linux-manager, especially https://github.com/QubesOS/qubes-desktop-linux-manager/blob/master/rpm_spec/qubes-desktop-linux-manager.spec.in#L90 (and the `touch` call in %post)